apps/condo/domains/organization/components/Recipient/SettingsContent.tsx

```diff
@@ -0,0 +1,83 @@
+import get from 'lodash/get'
+import { useIntl } from '@condo/next/intl'
+
+import React from 'react'
+import { Row, Col, Typography, Alert } from 'antd'
+import styled from '@emotion/styled'
+import { Gutter } from 'antd/es/grid/row'
+import { colors } from '@condo/domains/common/constants/style'
+
+import { useOrganization } from '@condo/next/organization'
+import { Recipient } from '@condo/domains/organization/components/Recipient'
+
+import { BillingRecipient, BillingIntegrationOrganizationContext } from '@condo/domains/billing/utils/clientSchema'
+
+
+const MEDIUM_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 40]
+
+const StyledAlert = styled(Alert)`
+  background-color: ${colors.warningAlert};
+
+  & > .ant-alert-description > div {
+    overflow: hidden;
+  }
+  
+  & > .ant-alert-content > .ant-alert-message {
+   color: ${colors.black}; 
+  }
+`
+
+export const RecipientSettingsContent = () => {
+    const intl = useIntl()
+    const PaymentsDetailsTitle = intl.formatMessage({ id: 'PaymentDetails' })
+    const AlertTitle = intl.formatMessage({ id: 'pages.condo.settings.recipient.alert.title' })
+    const AlertContent = intl.formatMessage({ id: 'pages.condo.settings.recipient.alert.content' })
+
+    const userOrganization = useOrganization()
+    const userOrganizationId = get(userOrganization, ['organization', 'id'])
+
+
+    //TODO(MAXIMDANILOV): DOMA-3252 go from BillingRecipient to Index
+    const {
+        obj: context,
+    } = BillingIntegrationOrganizationContext.useObject({
+        where: { organization: { id: userOrganizationId } },
+    })
+
+    const contextId = get(context, ['id'], null)
+
+
+    const {
+        loading: isRecipientsLoading,
+        objs: recipients,
+    } = BillingRecipient.useObjects({
+        where: {
+            context: { id : contextId },
```

Comment:
Admin and support users will see all entities at the front end regardless of the current organization. Is it OK?

Reply:
Not sure about the logic, but context has organiztion field, thus entities will be filtered by org